resource-validator/src/main/scala/com/broadinstitute/dsp/resourceValidator/ErroredRuntimeChecker.scala

```diff
@@ -13,12 +13,12 @@ import org.broadinstitute.dsde.workbench.model.TraceId
 object ErroredRuntimeChecker {
   def iml[F[_]: Timer](
     dbReader: DbReader[F],
-    deps: RuntimeCheckerDeps[F]
+    deps: BillingDeps[F]
```

Comment:
just curious, any reason why cloudBilling wasn't added to `RuntimeCheckerDeps` directly?

Reply:
It could have been. I did it this way originally to separate out the google1 dep, but thats no longer relevant. I like the idea of keeping runtimeCheckerDeps concise here though, its a pretty specific dep I don't think much else in the resource validator will care about